selfdrive/car/hyundai/carcontroller.py

```diff
@@ -78,7 +78,7 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, visual_alert,
 
     # 20 Hz LFA MFA message
     if frame % 5 == 0 and self.car_fingerprint in [CAR.SONATA, CAR.PALISADE, CAR.IONIQ, CAR.KIA_NIRO_EV, CAR.KONA_EV,
-                                                   CAR.IONIQ_EV_2020, CAR.IONIQ_PHEV, CAR.KIA_CEED, CAR.KIA_SELTOS, CAR.ELANTRA_2021, CAR.ELANTRA_HEV_2021]:
+                                                   CAR.IONIQ_EV_2020, CAR.IONIQ_PHEV, CAR.KIA_CEED, CAR.KIA_SELTOS, CAR.ELANTRA_2021, CAR.ELANTRA_HEV_2021,CAR.SONATA_HYBRID]:
```

Comment:
Instead of `HYBRID`, should we stick with the `HEV` convention that's used elsewhere?

Reply:
I want to update them all to hybrid anyway to match the other makes

Reply:
Ah alright, thanks. Figured we should keep it consistent. If we want to change the others instead, that makes sense too.